reference/catalog.v3.yml

```diff
@@ -2673,17 +2673,17 @@ paths:
                         The video type (a short name of a host site).
                       enum:
                         - youtube
-                  description: Common ProductVideo properties.
-                - type: object
-                  properties:
+                - properties:
                     video_id:
+                      type: string
                       maxLength: 25
                       minLength: 0
-                      type: string
                       description: |
                         The ID of the video on a host site.
                       x-required:
                         - post
+                      example: 'https://www.youtube.com/watch?v=`R12345677`'
```

Comment:
The video is no longer available.

Reply:
I'm having trouble finding a video that's under 25 characters

Reply:
Is it supposed to be a string of numbers?

Reply:
@bc-andreadao oh okay, I think I see.

Reply:
@bc-andreadao The `video_id` is the string that comes after the `v` query param in the YouTube link. So if the video URL is `https://www.youtube.com/watch?v=z3fRu9pkuXE`, one would just need to enter `z3fRu9pkuXE`.

Here is a full request body:

```json
{
  "title": "Writing Great Documentation",
  "description": "A video about documenation",
  "sort_order": 1,
  "type": "youtube",
  "video_id": "z3fRu9pkuXE"
}
```

Reply:
thanks, added in the example